client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-list-renderer.scss

```diff
@@ -1,3 +1,27 @@
 .pattern-list-renderer__pattern-list-item {
 	padding: 0;
+	position: relative;
+	overflow: visible;
+
+	.components-tooltip {
+		top: 100% !important;
+		left: 0 !important;
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)
I'd like to know the background of why these `top` and `left` are used 🙂 
Isn't it enough to pass `position="bottom center"` to `Tooltip`?

Reply:
+1 and I guess we can just use [default position](https://github.com/WordPress/gutenberg/blob/833176a4eefde095a4dee0a7397f622a82fd7479/packages/components/src/tooltip/index.js#L120) 🤔

Reply:
The default position doesn't work in the assembler. Or I couldn't make it work. Give it a try :)